index.html

```diff
@@ -237,12 +283,22 @@ <h5 class="mb">Would you like to know more information on bentos around you?</h5
 
 		    $(window).scroll(function () {
 		        EasyPeasyParallax();
+		        if ($('#bento-view-info').isOnScreen() && !isCountUpAnimated) {
+		        	console.log("Hey Buddy")
+		        	// get mixpanel event info
+		        	if (isViewDataSet) {
+				 	   $('#bento-view-info').countUpAnimate(0, currentView);
+				    	isCountUpAnimated = true;		        		
```

Comment:
The variable name was poor. It should be isBentoViewAnimated or something. So I put it outside the function in case we want to use this function with other info. in the future.